elrond-wasm/src/lib.rs

```diff
@@ -29,3 +29,6 @@ pub mod types;
 pub use hex_call_data::*;
 pub use io::*;
 pub use storage::{storage_clear, storage_get, storage_get_len, storage_set};
+
+#[cfg(feature="hex")]
```

Comment:
Since we need the hex_literal crate in the framework, there is no need to guard it by a feature.
Other than that great idea, we will include it in the next release one way or another.
We also want to add some macro for hard-coded addresses, since it is sometimes useful.